kotsadm/web/src/components/gitops/GitOpsDeploymentManager.jsx

```diff
@@ -113,21 +101,57 @@ class GitOpsDeploymentManager extends React.Component {
     }
   }
 
+  getGitopsState = async () => {
+    const freshGitops = await this.getGitops()
+
+    if (freshGitops.enabled) {
+      const selectedService = find(SERVICES, service => service.value === freshGitops.provider);
+      this.setState({
+        selectedService: selectedService ? selectedService : this.state.selectedService,
+        hostname: freshGitops.hostname || "",
+        gitops: freshGitops
+      })
+    } else {
+      this.setState({
+        gitops: freshGitops
```

Comment:
should add initial value like `gitops: {}` to the state here https://github.com/replicatedhq/kots/pull/982/files#diff-47d43402b26a6d7686c467ce611e0197R62